app_base.html

```diff
@@ -0,0 +1,47 @@
+<%inherit file = "/homepage/templates/app_base.htm" />
+<%! from catalog import models as cmod%>
```

Comment:
it might be close to put this right before you call cmod.Category.objects.all() so it's a little bit more clear

Reply:
Good call, thanks.